explainers/explainers/morris_sensitivity_explainer.py

```diff
@@ -38,7 +48,7 @@ class MorrisSensitivityLeExplainer(CustomExplainer, CustomDaiExplainer):
     # declaration of explanation types this explainer creates e.g. feature importance
     _explanation_types = [GlobalFeatImpExplanation]
     # Python package dependencies (can be installed using pip)
-    _modules_needed_by_name = ["gevent==1.5.0", "interpret==0.1.20"]
+    _modules_needed_by_name = ["interpret"]
```

Comment:
what is the guarantee that this recipe will work with any new version of `interpret`?
IMO it is best to fix to a specific version we tested against, since this recipe is built in DAI.

Reply:
@achraf-mer 
When I fix it to a specific version, DAI only installs the latest version version of `interpret`; that's why I didn't fix it to a particular version.

Reply:
fixed to the latest version